Monika After Story/game/script-affection.rpy

```diff
@@ -603,9 +603,8 @@ init 15 python in mas_affection:
 
         # unlocks wardrobe if we have more than one clothes available
         if len(mas_selspr.filter_clothes(True)) > 1:
-            store.mas_unlockEventLabel("monika_clothes_select")
-            #TODO: Amend monika_outfit if > 1 outfit available.
-            store.mas_lockEventLabel("monika_outfit")
+            store.mas_unlockEVL("monika_clothes_select", "EVE")
+            store.mas_unlockEVL("monika_outfit", "EVE")
```

Comment:
Should probably also unlock this for those who hypothetically got to Love, then fell below too. Since the topic covers all cases.

Reply:
there is no flag that specifies whether or not someone has reached love. We only know if they are currently in it.

Reply:
Indeed, but can just unlock it via update script or something, since then it'll just make sure it's done for everyone. Since either way, the topic is *still random* despite you not being able to get to it as it is currently.